### DISCUSSION TITLE
提问须知

### DISCUSSION THREAD

**Discussion by Night-stars-1** [0-8-4/miui-auto-tasks#227] [展示和讲述 ｜Show and tell]
1. 请不要发送以下样式的log，这样的log没有辨识度，我们会直接关闭或者锁定该反馈
```
�[36msystem_info�[0m.�[36msystem_info�[0m:�[36m30�[0m - �[1m
```
2. 反馈bug请发送`logs`文件夹下的log文件，不要直接复制控制台的输出内容
3. 请不要讨论如何获取token这里不会有人回答，请去其他地方讨论。
4. 请不要留任何平台的社交账号，该行为会锁定该反馈

**Comment by cy28438981** [comment]
请问下，其他地方，有组织没
